### ISSUE TITLE
How to create static pages

### ISSUE THREAD

**Issue by MiltonLn** [APSL/puput#117]
Hello, i'm new to Wagtail, puput and CMS in general.

But i have ready my blog, i just need to add a contact and about us static pages but i can't figure how to achieve this. I have reviewed on documentation and wagtail docs too.

I'm trying to add a django app with urls and also linking them on my ROOT_URLCONF but always i go to /about it's raising a 404 error by `wagtail.wagtailcore.views.serve` i taught this was not hard to do.

Am i doing something wrong? if not, how can i create static pages?

**Comment by sabueso** [comment]
Hi Milton

Seems rare to read this from you, due the fantastic work you've done in the Puput branch in your git profile (that's what I'm using now in prod in my personal page) ;)

In  http://ramiro.io I have barely the same you need: some static pages and the blog.
If I review the config of my wagtail project I'm using this:

`
INSTALLED_APPS = [
    #puput requirements
    'puput', 
    'compressor', 
    'wagtail.contrib.wagtailsitemaps',
    'wagtail.contrib.wagtailroutablepage',

    #all the rest ;)
    'home',
    'search',

    'wagtail.wagtailforms',
    'wagtail.wagtailredirects',
    'wagtail.wagtailembeds',
    .....
]

# Wagtail settings
WAGTAIL_SITE_NAME = "whateveryouwant"
# Base URL to use when referring to full URLs within the Wagtail admin backend -
# e.g. in notification emails. Don't include '/admin' or a trailing slash
BASE_URL = 'http://ramiro.io'
PUPUT_AS_PLUGIN =  False
PUPUT_USE_STREAM_BODY =  True
`

Basically, the idea, is to set-up a Wagtail site as mentioned in the Wagtail base documentation, create some models/static pages in Wagtail, and then, integrate the Puput app in the Wagtail project.

Once you have Puput setted up in Wagtail, in the admin website, you can create a child page for your app (in my case called "Home") with the type "Blog" and then, set up your blog in the desired URL, and create all the posts as childs pages from the main "Blog" page.

I hope guides you with your problem.

**Comment by MiltonLn** [comment]
Hello @sabueso,

Thanks for your help! 
It seems that my problem is that i've done the Puput installation as "Standalone blog app"
I will try your way and then close the issue if solved.

Thanks a lot

**Comment by marctc** [comment]
Thanks @sabueso for you help :+1:
